### DISCUSSION TITLE
Failure with new device support after creating PR

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#25280] [Devices]
Dear all,

with PR (https://github.com/Koenkk/zigbee-herdsman-converters/pull/8197) i create a new device support for the Philips Hue solo lightstrip (5 mtr.)  and @Koenkk merged it to the master.

After that and now with zigbee-Version 1.42.0-dev I do
- delete the device in z2m
- restart z2m
- set the device in pairing mode
- pairing the device with z2m successfull

But, it always show as unsupported device and i don't understand why?!?!

The function "generate external definition" create this for this device

```
const {philipsLight} = require('zigbee-herdsman-converters/lib/philips');

const definition = {
    zigbeeModel: ['LCL009'],
    model: 'LCL009',
    vendor: 'Signify Netherlands B.V.',
    description: 'Automatically generated definition',
    extend: [philipsLight({"colorTemp":{"range":[153,500]},"color":{"modes":["xy","hs"],"enhancedHue":true}})],
    meta: {},
};

module.exports = definition;
```

Where is my failure?

TIA and best regards

Gerd

**Comment by Koenkk** [comment]
Try updating to the latest-dev again, in the frontend -> about page the zigbee-herdsman-converters version should be at least `21.7.0`.
